### ISSUE TITLE
"Invalid date" error - location config

### ISSUE THREAD

**Issue by roest01** [roest01/docker-speedtest-analyser#8]
> Hi Tobias,
Thank you for contributing your analyses. i am running this on my Synology NAS and it looks great but one problem: I think the European date format isn’t working right for me in the US. When I look at the chart of results, the time axis simply says “Invalid date” for all values. I apologize for reaching out directly, but I think this must be a simple fix. Could you help?
> 
> Jon

![ea7e35ce-6106-44e1-9e81-2936e48d285a](https://user-images.githubusercontent.com/2870718/34715195-0db11cf6-f52c-11e7-9d35-faadd79add1c.png)

**Comment by roest01** [comment]
> You can configure the visualization frontend via volume in /var/www/html/js/appConfig.js

making appConfig.js editable is a good idea but in current implementation not well-thought-out.
Mounting a volume on a file produces problems, it's a lot easier to mount a volume on a folder.

**Do we still need appConfig.js editable ?**

reasonable to edit in current appConfig.js:

```
{
"dateFormat": "DD.MM.YYYY",
  "locale": "de",
  "labels": {
    "download": "Download",
    "ping": "Ping",
    "upload": "Upload"
  }
}
```

unreasonable to edit
```
{
 "daterange": {
     "timePicker": true,
     "timePicker24Hour": true,
     "startDate": moment().startOf('day'),
     ....
 },
 "labels": {
     ....
 }
}
```
I'd like to have an default appConfig with daterange config
and make selected options configurable only.  

There are some options:
1. Config via environment variables
2. automatic location identify (?)
3. Configure via interface (settings)
4. keep current idea and create appConfig.overwrite.js in a folder

**Any opinions?**

_notice: add documentation or link about valid `dateFormat` values_

**Comment by roest01** [comment]
solved in #13 - you can copy js/appConfig.example.js into data/appConfig.js and modify dateFormat and range values.
